mpisppy/cylinders/spcommunicator.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, spbase_object, fullcomm, intercomm, intracomm, options=None):
         self.fullcomm = fullcomm
         self.intercomm = intercomm
         self.intracomm = intracomm
-        self.rank_global = fullcomm.Get_rank()
+        self.global_rank = fullcomm.Get_rank()
         self.rank_inter = intercomm.Get_rank()
```

Comment:
Maybe we need to change `rank_intra` and `rank_inter` to `cylinder_rank` and `strata_rank`?

Reply:
Fixed